apps/api/src/bootstrap.ts

```diff
@@ -98,7 +98,7 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
 
   app.use(compression());
 
-  await setupSwagger(app);
+  // await setupSwagger(app);
```

Comment:
This needs to be addressed ASAP in a follow up PR. Dynamically setting the controller path in the `NovuModule` isn't compatible with Swagger, we need to ignore the dynamic path in the Novu NestJS handler.

Reply:
Yes that's my thinking. I will test it now to see if it's a quick solve.

Reply:
Resolved by explicitly declaring a custom injected controller.